piglow/examples/strobe.go

```diff
@@ -16,42 +13,24 @@ func main() {
 		panic(err)
 	}
 
-	// catch signals and terminate the app
```

Comment:
I have to admit that it is a better example. It's easier to read, it exits cleanly and doesn't have to use a signal. Note that the signal handling can be documented in the overall README or wiki (I do think it's something real users will want to want to learn about, but I now see that it doesn't have to be in the default examples).